### ISSUE TITLE
Use of 1/2 in a field is confusing

### ISSUE THREAD

**Issue by eriktaubeneck** [private-attribution/i-d#17]
It would probably be clearer to use $2^{-1}$, and maybe even define it, e.g., "... subtracting the multiplicative inverse of 2, $2^{-1}$, ..."

**Comment by martinthomson** [comment]
Yeah, we have a note at some point in the document saying what 1/2 actually is.  I thought that was fine, but Ben and I didn't really settle on anything.

It might be worth adding to that note.
